python/lsst/ip/diffim/subtractImages.py

```diff
@@ -180,6 +237,8 @@ def run(self, template, science, sources):
             Identified sources on the science exposure. This catalog is used to
             select sources in order to perform the AL PSF matching on stamp
             images around them.
+        **kwargs
+            Any additional keyword parameters.
```

Comment:
What is `**kwargs` for here? We don't use any other args in this `run()` method and we don't call `super` (and won't, be cause the super for pipeTask is not implemented), so we shouldn't accept any.